toolkit/primitives/domain/src/lib.rs

```diff
@@ -452,6 +452,16 @@ impl From<ecdsa::Public> for SidechainPublicKey {
 	}
 }
 
+/// Hex-encoded transaction CBOR (with or without 0x prefix). Represents CSL Transaction type
+#[derive(Clone, Encode, Decode, TypeInfo, PartialEq, Eq, Hash)]
+#[byte_string(debug, hex_serialize, hex_deserialize, decode_hex)]
+pub struct TransactionCbor(pub Vec<u8>);
+
+/// Hex-encoded VKeyWitness CBOR (with or without 0x prefix). Represents CSL VKeyWitness type.
```

Comment:
Same as above. This can wrap bytes (they don't have 0x prefix, hex string can have it) of VkeyWitness without CSL begin used.